StoreManagementService/src/PBJ.StoreManagementService.DataAccess/Entities/User.cs

```diff
@@ -12,18 +12,14 @@ public class User : BaseEntity
 
         public DateTime? BirthDate { get; set; }
 
-        public string? Login { get; set; }
+        public string? Email { get; set; }
 
-        public ICollection<Post>? Posts { get; set; }
+        public IReadOnlyCollection<Post>? Posts { get; set; }
 
-        public ICollection<Comment>? Comments { get; set; }
+        public IReadOnlyCollection<Comment>? Comments { get; set; }
```

Comment:
What is the reason to use IReadonlyCollection in entities? Are you assume that you will change those related entities only in specific repositories?

Reply:
Sure